docs/fidesops/docs/guides/saas_config.md

```diff
@@ -0,0 +1,370 @@
+```yaml
+test_request:
+  path: /3.0/lists
+```
+#### Endpoints
+This is where we define how we are going to access and update each collection in the corresponding Dataset. The endpoint section contains the following members:
+
+- `name` This name corresponds to a Collection in the corresponding Dataset.
+- `requests` A map of read and update requests for this collection. Each collection can define a way to read and a way to update the data. Each request is made up of:
+    - `path` A static or dynamic resource path. The dynamic portions of the path are enclosed within angle brackets `<dynamic_value>` and are replaced with values from request_params.
+    - `request_params`
+        - `name` Used as the key for query param values, or to map this param to a value placeholder in the path.
+        - `type` Either "query" or "path".
+        - `references` This is the same as reference in the Dataset schema. It is used to define the source of the value for the given request_param.
+        - `identity` This denotes the identity value that this request_param should take.
+        - `default_value` Hard-coded default value for a request_param. This is most often used for query params since a static path param can just be included in the `path`.
+    - `postprocessors` An optional list of response post-processing strategies. We will ignore this for the example scenarios below but an in depth-explanation can be found under [SaaS Post-Processors](saas_postprocessors.md)
+
+## Example scenarios
+#### Dynamic path with dataset references
+```yaml
+endpoints:
+  - name: messages
+    requests:
+      read:
+        path: /3.0/conversations/<conversation_id>/messages
+        request_params:
+          - name: conversation_id
+            type: path
+            references:
+              - dataset: mailchimp_connector_example
+                field: conversations.id
+                direction: from
+```
+In this example, we define `/3.0/conversations/<conversation_id>/messages` as the resource path for messages and define the path param of `conversation_id` as coming from the `id` field of the `conversations` collection. A separate GET HTTP request will be issued for each `conversations.id` value.
+
+```yaml
+# For three conversations with IDs of 1,2,3
+GET /3.0/conversations/1/messages
+GET /3.0/conversations/2/messages
+GET /3.0/conversations/2/messages
+```
+
+#### Identity as a query param
+```yaml
+endpoints:
+  - name: member
+    requests:
+      read:
+        path: /3.0/search-members
+        request_params:
+          - name: query
+            type: query
+            identity: email
+```
+In this example, the `email` identity value is used as a query param named "query" and would look like this:
```

Comment:
It's a shame Mailchimp has to use `query` here. It'd be less confusing if `query` wasn't used twice.